### ISSUE TITLE
Update 0.13.3 -> 0.14.0 shows CommandConnectorError during installation (database backup)

### ISSUE THREAD

**Issue by JustusRijke** [inventree/InvenTree#6629]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

While updating my 0.13.3 inventree, an error pops up:

```
# inv update|   Attempting uninstall: django-dbbackup
# inv update|     Found existing installation: django-dbbackup 4.0.2
# inv update|     Uninstalling django-dbbackup-4.0.2:
# inv upERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
django-q 1.3.9 requires redis<4.0.0,>=3.5.3, but you have redis 5.0.1 which is incompatible.
/opt/inventree/env/lib/python3.9/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")
Backing Up Database: inventree
CommandConnectorError: Error running:  pg_dump --dbname=postgresql://inventree:inventree@localhost:5432/inventree --format=custom
pg_dump: error: connection to database "inventree" failed: could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 5432?
could not connect to server: Cannot assign requested address
        Is the server running on host "localhost" (::1) and accepting
        TCP/IP connections on port 5432?

  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/utils.py", line 120, in wrapper
    func(*args, **kwargs)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/management/commands/dbbackup.py", line 93, in handle
    self._save_new_backup(database)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/management/commands/dbbackup.py", line 106, in _save_new_backup
    outputfile = self.connector.create_dump()
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/base.py", line 92, in create_dump
    return self._create_dump()
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/postgresql.py", line 112, in _create_dump
    stdout, stderr = self.run_command(cmd, env=self.dump_env)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/base.py", line 171, in run_command
    raise exceptions.CommandConnectorError(

Traceback (most recent call last):
  File "/opt/inventree/InvenTree/manage.py", line 24, in <module>
    main()
  File "/opt/inventree/InvenTree/manage.py", line 20, in main
    execute_from_command_line(sys.argv)
  File "/opt/inventree/env/lib/python3.9/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/opt/inventree/env/lib/python3.9/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/inventree/env/lib/python3.9/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/inventree/env/lib/python3.9/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/utils.py", line 120, in wrapper
    func(*args, **kwargs)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/management/commands/dbbackup.py", line 93, in handle
    self._save_new_backup(database)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/management/commands/dbbackup.py", line 106, in _save_new_backup
    outputfile = self.connector.create_dump()
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/base.py", line 92, in create_dump
    return self._create_dump()
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/postgresql.py", line 112, in _create_dump
    stdout, stderr = self.run_command(cmd, env=self.dump_env)
  File "/opt/inventree/env/lib/python3.9/site-packages/dbbackup/db/base.py", line 171, in run_command
    raise exceptions.CommandConnectorError(
dbbackup.db.exceptions.CommandConnectorError: Error running:  pg_dump --dbname=postgresql://inventree:inventree@localhost:5432/inventree --format=custom
pg_dump: error: connection to database "inventree" failed: could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 5432?
could not connect to server: Cannot assign requested address
        Is the server running on host "localhost" (::1) and accepting
        TCP/IP connections on port 5432?

date|       Successfully uninstalled django-dbbackup-4.0.2
# inv update|   Attempting uninstall: django-cors-headers
# inv update|     Found existing installation: django-cors-headers 4.3.0
# inv update|     Uninstalling django-cors-headers-4.3.0:
# inv update|       Successfully uninstalled django-cors-headers-4.3.0
```

The update finished, and InvenTree runs 0.14.0 afterwards, However, `Error object` is shown in the notification section. For this I will create a separate issue.

### Steps to Reproduce

run `apt upgrade`

### Expected behaviour

No errors

### Deployment Method

- [X] Docker
- [ ] Bare metal

### Version Information

# Version Information:
InvenTree-Version: 0.14.0
Django Version: 4.2.10
Commit Hash: f4fd84f5
Commit Date: None
Commit Branch: 0.14.x
stable
Database: postgresql
Debug-Mode: False
Deployed using Docker: False
Platform: Linux-5.10.0-27-amd64-x86_64-with-glibc2.31
Installer: PKG
Target: ubuntu:20.04
Active plugins: []

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

**Comment by SchrodingersGat** [comment]
@JustusRijke see comments in https://github.com/inventree/InvenTree/issues/6630 - please wait until `0.14.1` is available and then try :)

**Comment by SchrodingersGat** [comment]
@JustusRijke this should be addressed now - please comment here if your issues have not been solved

**Comment by JustusRijke** [comment]
I ran `apt upgrade` again on my 0.14.0 server (based on package `0.14.0-1709339795.f4fd84f5.focal`).
This installed a newer inventree package `0.14.0-1709585071.f4fd84f5.focal`. 
This package still has the same version number (0.14.0), but it does seem to fix #6630 and #6631 👍

**Comment by SchrodingersGat** [comment]
@JustusRijke great, thanks for reporting back!
